application/blocks/bestitamazonpay4oxid_order_confirm_button.tpl

```diff
@@ -51,40 +51,46 @@
                 var $form = $(this);
                 var $classInput = $('input[name="cl"]', $form);
                 var $functionInput = $('input[name="fnc"]', $form);
+                var $agbCheckbox = $('#checkAgbTop');
 
                 if ($classInput.val() === 'order'
                     && $functionInput.val() === '[{$oView->getExecuteFnc()}]'
                 ) {
                     $form.on('submit', function(e) {
-                        e.preventDefault();
-                        modalLoading.init(true);
-                        OffAmazonPayments.initConfirmationFlow(
-                            '[{$oViewConf->getAmazonConfigValue('sAmazonSellerId')}]',
-                            '[{$smarty.session.amazonOrderReferenceId}]',
-                            function (confirmationFlow) {
-                                $.ajax({
-                                    url: "/index.php",
-                                    data: {
-                                        cl: "order",
-                                        fnc: "confirmAmazonOrderReference",
-                                        stoken: "[{$oViewConf->getSessionToken()}]",
-                                        formData: $('#' + $form[0].id).serialize()
-                                    },
-                                    success: function (data) {
-                                        if (data.success === true) {
-                                            confirmationFlow.success();
-                                        } else {
-                                            window.location = data.redirectUrl;
-                                        }
-                                    },
-                                    error: function (data) {
-                                        confirmationFlow.error();
-                                        window.location = '/index.php?cl=user&fnc=cleanAmazonPay';
-                                    },
-                                    timeout: 5000
-                                });
-                            }
-                        );
+                        var agbCheckboxChecked = $agbCheckbox.length === 1 && $agbCheckbox.is(':checked');
```

Comment:
If you change above to $('input[name="ord_agb"]', $form) length should be checked to >= 1 and if any of it is :checked.

Reply:
Changed it